statsmodels/regression/heckman.py

```diff
@@ -0,0 +1,666 @@
+"""
+Heckman correction for sample selection bias (the Heckit procedure).
+
+Created August 19, 2014 by B.I.
+Last modified September 02, 2014 by B.I.
+
+NO warranty is provided for this software.
+"""
+
+import numpy as np
+import statsmodels.api as sm
+import statsmodels.base.model as base
+from statsmodels.iolib import summary
+from statsmodels.tools.numdiff import approx_fprime
+from scipy.stats import norm
+from scipy.stats import t
+import warnings
+import pandas as pd
+import pdb
+
+class Heckman(base.LikelihoodModel):
+    """
+    Class for Heckman correction for sample selection bias model.
+
+    Attributes
```

Comment:
the section header should be `Parameters` here, as explanation for `__init__` arguments, even though they are also attached.